### DISCUSSION TITLE
Compare by unique key

### DISCUSSION THREAD

**Discussion by houdelou** [WinMerge/winmerge#1856] [Q&A] [answered]
Hi,

Is there a way to compare a CSV file by using a unique key column ?

For example

**File 1**
1,cat
2,dog

**File 2**
1,cat
3,dog

Is there a way Winmerge could say : "Record with id 2 is missing from file 2 and record with id 3 is missing in file 1" ?

Thanks

**Comment by sdottaka** [comment, marked as answer]
Unfortunately, WinMerge doesn't have such a feature.

**Reply by houdelou** [reply]
Ok, thanks. Maybe it is not the purpose of the tool but I had to ask because I need such a tool. Thanks for the quick reply.
